test/src/receiver/main.cpp

```diff
@@ -16,45 +40,82 @@
 #include "../../../include/message.hpp"
 #include "../../../lib/vendor/include/NDTime.hpp"
 #include "../../../lib/vendor/include/iestream.hpp"
-#include "../../../include/receivercadmium.hpp"
+#include "../../../include/receiver_cadmium.hpp"
 
 using namespace std;
 using hclock=chrono::high_resolution_clock;
 using TIME = NDTime;
 
+/**
+ *  Sets input ports for message
+ */
+
+struct input : public cadmium::in_port<message_t>{};
 
-/***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct input : public cadmium::in_port<Message_t>{};
+/**
+ *  Sets Output ports for message
+ */
 
-/***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct output : public cadmium::out_port<Message_t>{};
+struct output : public cadmium::out_port<message_t>{};
 
+/**
+ * \brief class ApplicationGen for Application Generator.
+ *
+ *  The below class application generator(ApplicationGen) takes the file path
+ *  and transmits as message.
+ */
 
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
 template<typename T>
-class ApplicationGen : public iestream_input<Message_t,T>{
+class ApplicationGen : public iestream_input<message_t,T>{
     public:
+
+	/**
+     * Default constructor for the class
+	 */
+
     ApplicationGen() = default;
-    ApplicationGen(const char* file_path) : iestream_input<Message_t,
+
+    /**
+     * The below constructor of ApplicationGen class takes the input file path for
+     * the Application generator
+     */
+
+    ApplicationGen(const char* file_path) : iestream_input<message_t,
         T>(file_path) {}
-    };
+};
 
 
 int main(){
 
-    auto start = hclock::now(); //to measure simulation execution time
+	/**
+	 *  measures simulation execution time
```

Comment:
It is not measuring simulation execution time but initialize start variable to current time.